### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-General_Screen Reader: Narrator/NVDA is not reading the proper description of the "Action" menu button and tooltip provided for it, is not descriptive.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#156]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated:** MAS1.3.1, MAS1.1.1

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Server" in the left menu section and Press Enter.
5. Verify that Narrator is reading the correct description of the "Action" buttons
6. Verify that tooltip provided for "Action" menu button is descriptive.

**Actual Result:**
Narrator/NVDA is reading the Web Server Action menu button as just: "Action Buttons" which is not descriptive and screen reader users will be unable to understand that which action will be performed when they select this button. Even the tooltip provided for it is not descriptive and it is not giving an understandable context for the control.

**Expected Result:**
Screen Reader should read it as: "Web Server action menu button, select to start or stop the web server". Even tooltip should be descriptive in a way that gives users an additional context and help them understand the control properly.

**User Impact:**
Screen Reader users will be unable to understand that which kind of action will be performed when they will select this button. Even sighted users will be unable to understand this by seeing the tooltip provided.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Attachments:**
[Bug 156_description of action button is not proper.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331113/Bug.156_description.of.action.button.is.not.proper.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
